util/bootloader_atmega32a_1_0_0.hex

```diff
@@ -0,0 +1,154 @@
+:107000000C9468380C94A0380C9485380C9485380E

```

Comment:
Where wasthis bootloader pulled from?

Reply:
per https://github.com/qmk/qmk_firmware/pull/4914, https://blog.winkeyless.kr/m/152. 
Should i re-write so it gives info for people to grab it themselves instead (and rip it out of the repo also)?

Reply:
Okay, thanks.  We just wanted to ensure the chain of custody here.

Reply:
so keep as-is or rewrite?

Reply:
Hmm, I'm not sure what is best.  If we know the source, that should be okay, I think.  I'd like to see a link to it in the docs (it can be hidden, though).

@jackhumbert @skullydazed comments?

Reply:
I personally would prefer just a link.....but who knows how long winkeyless.kr will be up?

Reply:
i know that that hex file is in my dropbox now! (just in case, right?)